src/assets/text/droughtAnnotations_desktop.js

```diff
@@ -1,36 +1,95 @@
 export default {
     timelineEvents: [
-      {
-        id: '1970s_1', // Unique id for annotation - order and # value don't matter - annotations are later sorted by start_date
-        date: '1974-05-01', // Date marking start of annotation. Determines y trigger start position on deskop and order
-        text: 'The 1976-1977 drought was short, so it is not considered one of the five major drought events in the last 100 years. But this two-year drought caused agricultural losses and hydropower shortages across much of the western U.S.', // Narrative text
-        desktop_x_offset_per: 10,
-        desktop_text_anchor: 'start',
-        desktop_text_width: 300
-      },
       {
         id: '2',
         date: '1924-09-01',
-        text: 'Longest drought on record, lasted 705 days!',
+        text: 'Droughts are colored by duration where darker = longer',
         desktop_x_offset_per: 59,
         desktop_text_anchor: 'start',
-        desktop_text_width: 100
+        desktop_text_width: 200
       },
       {
         id: 'legend',
         date: '1922-06-01',
-        text: 'Droughts are colored by duration where darker = longer',
+        text: 'The longest drought was 2208 days',
```

Comment:
I still think this would work better on the righthand panel. 
<img width="975" alt="image" src="https://github.com/DOI-USGS/drought-timeline/assets/17803537/d831e40f-7401-4201-aed6-b5d7025e16be">
It's somewhat unclear that this is a legend. It needs more to guide the audience. For example, a small arrow or tick mark connecting the text to either end of the color scale. I wonder if a color scale made up of the bars would be more clear.

Reply:
When I land on the page, I first look at the title, then the legend, then the "how to read the timeline". Doing that, I miss that each line is a streamgage, which makes the legend confusing,

Required edits:
- [ ] fix awkward text wrapping - just "days" alone on a line
- [ ] make legend more informative
- [ ] add an x-axis label - "Number of streamgages in drought"
- [ ] fix arrow placement with text annotation

Reply:
The landing page now has a less distracting legend, more informative description in the "How to read this timeline" narration, and a description of y- and x- axes, map, and an example annotation for orientation. The annotation text is also lighter, so it doesn't compete as much with the information on the right. 

Here's a "smaller" screen on the left at landing, and another bigger screen on the right. 
![image](https://github.com/DOI-USGS/drought-timeline/assets/19958841/7ca438c4-d9ec-426c-8288-64c0502abeff)

Reply:
Text wrapping is also better, allowing for better readability and consistency with how arrows and annotations are oriented, regardless of screen size (at least based on my testing):

![image](https://github.com/DOI-USGS/drought-timeline/assets/19958841/ad7314c5-cfcb-4737-afbf-2e96a8bbc429)

Reply:
the map annotation looks like it goes with the chart, but it's really about the map